features/guidance/collapse.feature

```diff
@@ -344,7 +344,7 @@ Feature: Collapse
             | waypoints | route                   | turns                                      |
             | a,d       | first,first,first,first | depart,continue left,continue right,arrive |
             | a,e       | first,second,second     | depart,turn left,arrive                    |
-            | a,f       | first,third,third       | depart,turn straight,arrive                |
+            | a,f       | first,third,third       | depart,new name straight,arrive            |
```

Comment:
Why did this change? Seems like the original behavior of a "turn straight" made more sense in this context.